libats-messaging-datatype/src/main/scala/com/advancedtelematic/libats/messaging_datatype/DataType.scala

```diff
@@ -1,7 +1,32 @@
 package com.advancedtelematic.libats.messaging_datatype
 
+import java.time.Instant
+import java.util.UUID
+
+import akka.http.scaladsl.model.Uri
+import com.advancedtelematic.libats.data.Namespace
+import com.advancedtelematic.libats.data.UUIDKey.{UUIDKey, UUIDKeyObj}
+import eu.timepit.refined.api.{Refined, Validate}
+
 object DataType {
   case class PackageId(name: String, version: String) {
     def mkString = s"$name-$version"
   }
+
+  case class ValidCommit()
+  type Commit = Refined[String, ValidCommit]
+
+  implicit val validCommit: Validate.Plain[String, ValidCommit] =
+    Validate.fromPredicate(
+      hash => hash.forall(h => ('0' to '9').contains(h) || ('a' to 'f').contains(h)),
```

Comment:
Would it be worth checking the length of the hash in here also? SHA-256 hashes are 64 chars in length.